piyushgarg-dev/piyushgargdev-nextjs
# Fixes #22

## What does this PR do?
This change fixes a bug of the internal server error 500 after discord and Github login .

Fixes #22 

## Type of change


- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?
- Open the website locally http://localhost:3000
- Go to the Login tab 
- Login via Discord or Github. 
- You should be redirected back to the "home page" of the website.
